draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -118,9 +118,11 @@ wherever possible.
 ## Feature Summary
 
 * This specification defines
- - a format for an Issuer-signed JWT containing selectively disclosable claims that include object properties (key-value pairs), array elements, and nested data structures built from these,
-  - a format for data associated with an Issuer-signed JWT that enables selectively disclosing claims, and
-  - formats for the combined transport of an Issuer-signed JWT and the associated data during issuance and presentation.
+ - a format for the payload of an Issuer-signed JWT containing selectively disclosable claims that include object properties (key-value pairs), array elements, and nested data structures built from these,
+ - a format for data associated with the JWT that enables selectively disclosing those claims,
+ - facilities for binding the JWT to a key and associated data to prove possession thereof, and
```

Comment:
```suggestion
 - facilities for binding the Issuer-signed JWT to a key and associated data to prove possession thereof, and
```
what does `facilities` mean in this context?

Reply:
"facilities" is meant to mean like "ways of doing" or "mechanisms for"

Reply:
ok, cool.